web/components/auth-context.tsx

```diff
@@ -211,17 +206,6 @@ export function AuthProvider(props: {
     }
   }, [uid])
 
-  useEffect(() => {
-    if (authLoaded && uid) {
-      const privateUserListener = listenForPrivateUser(uid, (privateUser) => {
-        setPrivateUser(privateUser ?? undefined)
-      })
-      return () => {
-        privateUserListener()
-      }
-    }
-  }, [authLoaded, uid])
-
```

Comment:
Uh, shouldn't we replace this with polling in the same PR?

I think we could also figure out auth'd websockets by sending encrypted messages with asymmetric key encryption. Which apparently @mqp did for supabase channels before we switched to websockets.

Reply:
or we could just not send the info at all, and expect the client to refetch 🤔